astro/src/content/quickstarts/quickstart-php-laravel-api.mdx

```diff
@@ -0,0 +1,424 @@
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/Http/Controllers/ChangeBank/MakeChangeController.php"} lang="php"/>
+
+To finish, create a controller at `app/Http/Controllers/ChangeBank/PanicController.php` to handle the `/panic` endpoint for users with the `teller` role.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/Http/Controllers/ChangeBank/PanicController.php"} lang="php"/>
+
+### Validating Issuer and Audience Claims
+
+Besides verifying the signature, it is also recommended to validate the issuer (`iss`) and audience (`aud`) claims when receiving a JWT. To do those checks, you need to create two files.
+
+First, create a file named `app/FusionAuth/Claims/Audience.php` that will check if the audience (`aud`) claim actually contains the Client Id for the application you created in FusionAuth.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/FusionAuth/Claims/Audience.php"} lang="php"/>
+
+Now, create a file called `app/FusionAuth/Claims/Issuer.php` to check if the issuer (`iss`) is actually your FusionAuth instance address.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/FusionAuth/Claims/Issuer.php"} lang="php"/>
+
+### Let your Code Know about the FusionAuth Instance
+
+Now that you created all these files, you need to pass some details about the FusionAuth configuration to them.
+
+Start by adding the environment variables below to your `.env` file, changing `https://The-Address-Ngrok-Gave-You` to the actual address you copied from ngrok.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/.env"} tags="fusionauth" lang="ini"/>
+
+Now open the `config/app.php` file and browse to the `providers` section. This is a Laravel configuration to automatically load these files when booting your application. Add `App\FusionAuth\Providers\FusionAuthServiceProvider::class` to the end of the `providers` array, like shown below.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/config/app.php"} tags="providers" lang="php"/>
+
+In that same `config/app.php` file, add these lines at the end (just before the last line) to pass values from the environment variables to the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/config/app.php"} tags="fusionauth" lang="php"/>
+
+## Run the API
+
+### Get a Token
+
+There are [several ways to acquire a token](/docs/v1/tech/oauth/) in FusionAuth, but for this example you will use the [Login API](/docs/v1/tech/apis/login) to keep things simple.
+
+First let's try the requests as the <code><RemoteValue url={frontmatter.coderoot + "/kickstart/kickstart.json"} selector="$.variables.userEmail" /></code> user. Based on the configuration this user has the `teller` role and should be able to use both `/make-change` and `/panic`.
```

Comment:
I get `Not Available` user.
![Screenshot 2023-10-24 at 4 49 12 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/34e51312-0ac4-412b-a5f9-f47bdab29903)